docs/imviz/displayimages.rst

```diff
@@ -289,13 +311,34 @@ Min
 ^^^
 
 If the percentile is "custom", then the data value corresponding to the
-minimum of the colormap scale (e.g. black in grayscale) can be set.
+minimum of the colormap scale (e.g., black in grayscale) can be set.
 
 Max
 ^^^
 
 If the percentile is "custom", then the data value corresponding to the
-maximum of the colormap scale (e.g. white in grayscale) can be set.
+maximum of the colormap scale (e.g., white in grayscale) can be set.
+
+From the API
+^^^^^^^^^^^^
+
+To set the percentile for just the image being displayed::
+
+    viewer = imviz.default_viewer
+    viewer.cuts = '95%'  # Preset
+    viewer.cuts = (0, 1000)  # Custom
+
+To set the percentile for all the images at once::
+
+    plot_options.select_all()
+
+    # Preset
+    plot_options.stretch_preset = '95%'
+
+    # Custom
+    plot_options.stretch_preset = 'Custom'
```

Comment:
technically setting the preset to 'Custom' isn't necessary, but I agree its nice to be verbose here to show the user what is happening.

Reply:
I agree; I usually like examples where they show "a little extra"